.eslintrc.yml

```
@@ -0,0 +1,18 @@
+env:
+  browser: true
+  es6: true
+  mocha: true
+extends: 'eslint:recommended'
+globals:
+  Atomics: readonly
+  SharedArrayBuffer: readonly
+  expect: true
+  global: true
+  require: true
+  process: true
+  beforeAll: true
```

Does ET need "before"? - @elizabethsjudd is there an instance where the `before` (or `after`) function would be called or seen by eslint?

---

@scottnath @ChevonNwosu to the best of my knowledge this is the only place where we have hardcoded `mocha` code... however this file should probably not be linted as it's more documentation than anything.

---

This file could be linted by a yaml linter fwiw (out-of-scope for this pr tho)

my q was about the actual rule. Exported-tests [use `before` when it is not declared](https://github.com/IBM/exported-tests/blob/85c015ecc4fe3a46dda03fb1e7d09e73905fa786/src/parsers/tests/BDD.js#L108) so we should prol add that here as well

```suggestion
  beforeAll: true
  before: true
```

Once the linting is up and running, it will surface if there are any more like `before`.